### PR TITLE
Create New Solution and Playbook for VirusTotal

### PR DESCRIPTION
**Pull request by v-amolpatil**
Required items, please complete
   
   Change(s):
   - Created new solution and added playbooks and packaged.
FileHash Enrichment - Virus Total report
IP Enrichment - Virus Total report
URL Enrichment - Virus Total report
URL Enrichment - Virus Total domain report

   Reason for Change(s):
   - Need to have a new solution package and playbook support.

   Version Updated:
   - 1.0.0

   Testing Completed:
   - Yes, verified Custom deployment is working.

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

Solution type either 1P/3P/ISV is not specified anywhere in sheet.

### CODE REVIEW

**Comment by v-rucdu** [comment]
All the playbooks files are showing as Add instead of rename. Please fix it

File: Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json [hunk 1]
```diff
@@ -360,8 +386,8 @@
                 "[resourceId('Microsoft.Logic/workflows', parameters('PlaybookName'))]"
             ],
             "properties": {
-                "roleDefinitionId": "[concat('/subscriptions/', variables('ASSubscriptionId'), '/', variables('ASResourceGroup'), '/providers/Microsoft.Authorization/roleDefinitions/ab8e14d6-4a74-4a29-9ba8-549422addade')]",
-                "principalId": "[reference(concat(resourceId('Microsoft.Logic/workflows', parameters('PlaybookName')), '/providers/Microsoft.ManagedIdentity/Identities/default'), '2018-11-30').principalId]"
+                "roleDefinitionId": "[[variables('_roleDefinitionId')]",
```
Comment (v-rucdu) [inline Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json:389]: why are we adding an extra [ bracket in source file? please revert this change

File: Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json [hunk 2]
```diff
@@ -360,8 +386,8 @@
...
+                "principalId": "[[variables('_principalId')]"
```
Comment (v-rucdu) [inline Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json:390]: @v-amolpatil same as above, remove extra square bracket

**Review by v-rucdu — CHANGES_REQUESTED** [review]

File: Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json (hunk 1 above)
Comment (v-amolpatil) [reply Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json]: removed extra bracket

File: Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json (hunk 2 above)
Comment (v-amolpatil) [reply Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json]: done

**Review by v-rucdu — APPROVED** [review]
